### ISSUE TITLE
Scrolling no worky on touch-enabled devices

### ISSUE THREAD

**Issue by dvdotsenko** [polarblau/stickySectionHeaders#1]
On touch-enabled devices an EventObj.preventDefault() is needed on wrapper container for ontouchstart, ontouchmove
Instead of scrolling the list it scrolls the whole page.

**Comment by polarblau** [comment]
That's true. I'm planning on supporting touch enabled devices in a future release.
